### PR TITLE
Add banner for personal30 users

### PR DESCRIPTION
**Pull request by amartinj**
Closes [#1886](https://github.com/CartoDB/cartodb-central/issues/1886)
This is the PR containing the changes affecting cartodb.

### CODE REVIEW

**Comment by amartinj** [comment]
@elenatorro I've already added the banner for the static profile

File: app/views/admin/shared/_trial_notification.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<% if current_user.has_feature_flag?('no_free_tier') && current_user.account_type == 'PERSONAL30' %>
+    <div class="CDB-Text FlashMessage FlashMessage--main">
+        <div class="u-inner">
+            <div class="FlashMessage FlashMessage-info FlashMessage--main u-flex u-justifySpace u-alignCenter">
+                <p class="u-flex">You're currently under your 30-day Trial Period. Add your billing info to keep using our platform.</p>
```
Comment (nobuti) [inline app/views/admin/shared/_trial_notification.html.erb]: I feel we don't need `u-flex` here, but I may be wrong.

Comment (javitonino) [reply app/views/admin/shared/_trial_notification.html.erb]: You are right, removing!

File: app/views/admin/shared/_trial_notification.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<% if current_user.has_feature_flag?('no_free_tier') && current_user.account_type == 'PERSONAL30' %>
```
[the remark this answers is not in the document]
Comment (javitonino) [reply app/views/admin/shared/_trial_notification.html.erb]: Indeed, I'll change it real quick.

**Pull request merged**